### PR TITLE
AST-1854 - Added area label to link wave accessibility issue

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
1) Enable a related post on a single post and check if the content of the related post is large, there is showing (…) but this will not pass accessibility in the tool.

This is showing error in the E2E test case aria-label is missing.

### Screenshots
https://share.bsf.io/RBunbDGm

### Types of changes

 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Added in the card

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/modules/related-posts/class-astra-related-posts-markup.php [hunk 1]
```diff
@@ -200,7 +200,7 @@ public function astra_get_related_post_read_more( $current_post_id ) {
 
 		?>
 			<p class="ast-related-post-cta read-more">
-				<a class="ast-related-post-link <?php echo esc_html( $custom_class ); ?>" href="<?php echo esc_url( apply_filters( 'astra_related_post_link', get_the_permalink(), $current_post_id ) ); ?>" target="<?php echo esc_html( $target ); ?>" rel="bookmark noopener noreferrer"><?php echo esc_html( $cta_text ); ?></a>
+				<a class="ast-related-post-link <?php echo esc_html( $custom_class ); ?>" href="<?php echo esc_url( apply_filters( 'astra_related_post_link', get_the_permalink(), $current_post_id ) ); ?>" aria-label="ast-related-post-link"  target="<?php echo esc_html( $target ); ?>" rel="bookmark noopener noreferrer"><?php echo esc_html( $cta_text ); ?></a>
```
Comment (imnavanath) [inline inc/modules/related-posts/class-astra-related-posts-markup.php:203]:
```suggestion
				<a class="ast-related-post-link <?php echo esc_html( $custom_class ); ?>" href="<?php echo esc_url( apply_filters( 'astra_related_post_link', get_the_permalink(), $current_post_id ) ); ?>" aria-label="<?php echo esc_attr__( 'Related post link', 'astra' ); ?>"  target="<?php echo esc_html( $target ); ?>" rel="bookmark noopener noreferrer"><?php echo esc_html( $cta_text ); ?></a>
```

**Comment by imnavanath** [comment]
@GauravKhupse please add changelog for this.

File: inc/modules/related-posts/class-astra-related-posts-markup.php (hunk 1 above)
Comment (GauravKhupse) [reply inc/modules/related-posts/class-astra-related-posts-markup.php]: Changes done!

**Review by imnavanath — APPROVED** [review]

File: changelog.txt [hunk 2]
```diff
@@ -1,12 +1,15 @@
-v3.8.0 (Unreleased)
+v3.8.1 (Unreleased)
+- Fix: W3 validation - aria-label attribute is missing for releated posts link.
+- Fix: W3 validation - Role attribute is missing for Acoount element link.
```
Comment (imnavanath) [inline changelog.txt:3]: This is missing or already merged I guess.

Comment (GauravKhupse) [reply changelog.txt:3]: account link task is merged but the changelog is not updated properly, after this merge of this PR, it will show it properly.

**Pull request merged**